src/v1/base/LikeButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { APIBookshelfInfo } from '@/types/bookshelf';
+import { IconHeart } from '@public/icons';
+
+type LikeButtonProps = {
+  isLiked: APIBookshelfInfo['isLiked'];
+  likeCount: APIBookshelfInfo['likeCount'];
+  onClick: () => void;
+};
+
+const LikeButton = ({ isLiked, likeCount, onClick }: LikeButtonProps) => {
+  const BG_COLOR_CLASS = isLiked ? 'bg-warning-800' : 'bg-white';
+  const ICON_COLOR_CLASS = isLiked ? 'stroke-white' : 'stroke-warning-800';
+  const TEXT_COLOR_CLASS = isLiked ? 'text-white' : 'text-warning-800';
+
+  return (
+    <button
+      onClick={onClick}
+      className={`${BG_COLOR_CLASS} flex h-[2.4rem] w-[5.4rem] items-center gap-[0.4rem] rounded-full border-[0.1rem] border-warning-800 bg-warning-800 px-[0.8rem] py-[0.2rem]`}
+    >
+      <IconHeart
+        className={`${ICON_COLOR_CLASS} h-[1.3rem] w-[1.3rem] fill-white stroke-[0.15rem]`}
+      />
+      <p
+        className={`${TEXT_COLOR_CLASS} w-[2rem] text-center text-xs font-bold`}
+      >
+        {likeCount}
+      </p>
+    </button>
+  );
+};
```

Comment:
p1;

버튼의 padding 때문에 공간이 충분하지 않아서 아이콘이 스타일링한 크기보다 작게 보여지고 있는 것 같아요. 좌우 여백도 대칭일 수 있도록 마크업을 살짝 바꿨는데, 이렇게 수정해보면 어떨까요?! `min-w`를 줘서 좋아요 수가 많아지면 자연스럽게 width가 커질 수 있도록 구현했어요! (좋아요 숫자가 늘어나는데는 어느정도 한계가 있으니 말줄임을 하는 것보다 버튼의 크기가 늘어나는 것이 자연스럽다고 생각했어요.)

- 현재 마크업
<img width="432" alt="스크린샷 2023-12-11 오후 10 56 50" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/57716832/a194efe0-9ce7-4cdb-a4c2-ab2284d33b07">

- 수정한 마크업
<img width="410" alt="스크린샷 2023-12-11 오후 11 38 50" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/57716832/a16b2d49-7b7d-4415-a929-b11af6c887a5">

```suggestion
 <button
      onClick={onClick}
      className={`${BG_COLOR_CLASS} flex h-[2.4rem] min-w-[5.2rem] items-center justify-center gap-[0.4rem] rounded-full border-[0.1rem] border-warning-800 bg-warning-800 px-[1rem]`}
    >
      <IconHeart
        className={`${ICON_COLOR_CLASS} h-[1.5rem] w-[1.5rem] fill-white stroke-[0.15rem]`}
      />
      <p
        className={`${TEXT_COLOR_CLASS} min-w-[1.5rem] text-center text-xs font-bold`}
      >
        {likeCount}
      </p>
    </button>
```

Reply:
[91d3639](https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/pull/458/commits/91d363980475b209b5204ab2286877d9d5126c28)

수정하였습니다! 훨씬 좋은것 같아요!!
아이콘이 생각처럼 스타일링이 안되길래 뭐가 문제였나 싶었는데
수직 padding값 때문이였네요 👍